src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
```

Comment:
Reasoning for using `inline-flex`?
If it was flex, wouldnt need to apply 100% width in htmls...
I am wondering, if we could get rid of container.. Should be possible to place 3 elements in one line without it. Just then width 100% is not really correct value on the slider..

Reply:
I only chose it because I heard inline-flex was more efficient to render than flex.

Reply:
I think just flex would suit more here, so we ensure, that slider takes 100% of the parent width unless specified differently..
With inline-flex we would need to always add 100% width so the slider would work.
With `inline-flex`:
![image](https://user-images.githubusercontent.com/81580355/124917117-1f9bf800-dffc-11eb-957b-82967c2f0f5f.png)
With `flex`:
![image](https://user-images.githubusercontent.com/81580355/124917175-2e82aa80-dffc-11eb-9444-a59cc3c6888d.png)

Have in mind, I was using here slider WITHOUT input container wrapper.

Reply:
Switched to flex and removed occurances of width: 100% in html